### PR TITLE
Protected terms can be redefined, if identical

### PR DESCRIPTION
**Pull request by pchampin**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/pull/190.html" title="Last updated on Jun 10, 2019, 10:28 PM UTC (a63b99c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/190/22176e9...a63b99c.html" title="Last updated on Jun 10, 2019, 10:28 PM UTC (a63b99c)">Diff</a>

### CODE REVIEW

**Comment by gkellogg** [comment]
We need to add something to the changes section; otherwise, looks great.

File: index.html [hunk 1]
```diff
@@ -3650,14 +3650,115 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
...
+    (modulo the <code>@protected</code> flag).
+    The rationale is that the new definition does not violate the protection,
+    as it does not change the semantics of the protected term.
+    This is useful for widespread term definitions,
+    such as e.g. aliasing <code>@type</code> to <code>type</code>,
+    which may occur (including in a protected form) in several contexts.
+  </p>
+
+  <aside class="example ds-selector-tabs changed"
+  title="overriding permitted if both definitions are identical">
```
Comment (davidlehn) [inline index.html]: Capitalize "Overriding" here, and in the `data-result-for` bits.  The property scoped context example below has the same issue.

File: index.html [hunk 2]
```diff
@@ -3650,14 +3650,115 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
...
-    a property-<a>scoped context</a> is not affected by protection,
-    and can therefore override protected terms,
+    there are two exceptions to this rule.
+    The first exception is that a context is allowed to redefine a protected term
+    if the new definition is identical to the protected term definition
+    (modulo the <code>@protected</code> flag).
+    The rationale is that the new definition does not violate the protection,
+    as it does not change the semantics of the protected term.
+    This is useful for widespread term definitions,
+    such as e.g. aliasing <code>@type</code> to <code>type</code>,
```
Comment (davidlehn) [inline index.html]: "such as e.g." seems repetitive.  How about:
```suggestion
    such as aliasing <code>@type</code> to <code>type</code>,
```

File: index.html [hunk 3]
```diff
@@ -3650,14 +3650,115 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
...
+            "Organization": "http://example.org/orga/Organization",
+            "member": {
+              "@id": "http://example.org/orga/member",
+              "@type": "@id"
+            }
+          },
+          {
+            ****"id": "@id",
+            "type": "@type",****
+            ####­– Those "redefinition" do not raise an error.
```
Comment (davidlehn) [inline index.html]:
```suggestion
            ####­– Those "redefinitions" do not raise an error.
```
Also are the quotes needed?

File: index.html [hunk 4]
```diff
@@ -12690,7 +12791,9 @@ <h2>Changes since JSON-LD Community Group Final Report</h2>
       This allows a mechanism for documenting the content of a context using HTML.</li>
     <li>A <a>frame</a> may also be located within an HTML document, identified
       using type <code>application/ld+json;profile=http://www.w3.org/ns/json-ld#frame</code>.</li>
-  </ul>
+    <li>Term definitions can now be <a href="#protected-term-definitions">protected</a>,
+      to limit the ability of other contexts to override them.</li>
+    </ul>
```
Comment (davidlehn) [inline index.html]: Fix indent.
```suggestion
  </ul>
```

File: index.html (hunk 1 above)
Comment (gkellogg) [reply index.html]: Done.

**Comment by gkellogg** [comment]
Awaiting updates to https://github.com/w3c/json-ld-api/pull/92.

**Pull request merged**